packages/react-vapor/src/components/modal/ModalComposite.tsx

```diff
@@ -28,6 +28,7 @@ export interface IModalCompositeOwnProps
     modalFooterClasses?: IClassName;
     isPrompt?: boolean;
     validateShouldNavigate?: (isDirty: boolean) => boolean;
+    redirectOnClose?: () => void;

```

Comment:
I guess that you want to passe the redirection logic as a function in the props but the name of the props feels like it will receive a boolean value...